HomeWork_One/Quiz_Template/src/components/inputs/input.tsx

```diff
@@ -0,0 +1,14 @@
+import './Input.css'
+function Input() {
+    return(
+        <>
+            <div className='input-container'>
+                <label htmlFor="styled-input"  className="input-label">Number of questions</label>
+                <input type='number'  max='15' min='5' id="styled-input" placeholder="Enter number from 5 to 15" required/>
+                <span className="input-underline"></span>
+            </div>
+        </>
+    )
+}
```

Comment:
I recommend you update with component and make it more generic
e.g. if you need to change a label? if you need change a type or placeholder?
I think it would be a bad approach to create a new component for each specific case
WDYT?

Reply:
In this case you mean that i need  use props for all this thing?

Reply:
indeed